examples/share_encrypted_data_with_partners/encrypt_decrypt_sample.tf

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+* The resources in this file provides an usage example of
+* encrypt/decrypt process using python scripts samples.
+* For production use, use your company's approved method to encrypt/decrypt."
```

Comment:
I think the warning is not needed here, it's pretty clear that it's a sample and at least it doesn't pose serious security concerns, unlike generating weak random keys in some of the other samples. WDYT?
```suggestion
* The resources in this file provide a usage example of
* Cloud KMS-based encryption and decryption, using sample python scripts.
```

Reply:
Yeah, makes sense to me. Updated!